wren-ai-service/tools/config/config.example.yaml

```diff
@@ -107,7 +107,10 @@ pipes:
     document_store: qdrant
   - name: data_assistance
     llm: openai_llm.gpt-4o-mini
-
+  - name: sql_executor
+    engine: wren_ui
```

Comment:
should we put the sql executor in other config as well?

Reply:
I'll remove it as of now, since we don't use this at the moment